reference/5.1/Microsoft.PowerShell.Core/About/about_PSSnapins.md

```diff
@@ -50,40 +48,29 @@ To get the snap-in for each  Windows PowerShell provider, type:
 Get-PSProvider | Format-List name, pssnapin
 ```
 
+To get all the registered snap-ins on your system or to verify that a snap-in
+is registered, type:
+
+```powershell
+Get-PSSnapin -registered
+```
+
 To get a list of the cmdlets in a  Windows PowerShell snap-in, type:
 
 ```powershell
 Get-Command -Module <snap-in_name>
 ```
 
-### INSTALLING A SNAP-IN
+## Install a snap-in
 
 The built-in snap-ins are registered in the system and added to the default
 session when you start Windows PowerShell. However, you have to register
 snap-ins that you create or obtain from others and then add the snap-ins to
 your session.
 
-### REGISTERING A SNAP-IN
-
-A Windows PowerShell snap-in is a program written in a .NET Framework language
-that is compiled into a .dll file. To use the providers and cmdlets in a
-snap-in, you must first register the snap-in (add it to the registry).
-
-Most snap-ins include an installation program (an .exe or .msi file) that
-registers the .dll file for you. However, if you receive a snap-in as a .dll
-file, you can register it on your system. For more information, see
-[How to Register Cmdlets, Providers, and Host Applications](/previous-versions//ms714644(v=vs.85)).
-
-To get all the registered snap-ins on your system or to verify that a snap-in
-is registered, type:
-
-```powershell
-Get-PSSnapin -registered
-```
-
-### ADDING THE SNAP-IN TO THE CURRENT SESSION
+## Add a snap-in to the current session
 
-To add a registered snap-in to the current session, use the Add-PsSnapin
+To add a registered snap-in to the current session, use the `Add-PsSnapin`
 cmdlet. For example, to add the Microsoft SQL Server snap-in to the session,
```

Comment:
1. Do we use module formatting for snap-in names?
2. Is this snap-in still available and in use, or is it a module now?

Reply:
I don't think it matters for this context.